modes/yaml-mode/evil-collection-yaml-mode.el

```diff
@@ -0,0 +1,41 @@
+;;; evil-collection-yaml-mode.el --- Bindings for `yaml-mode' -*- lexical-binding: t -*-
+
+;; Author: Frédéric Giquel
+;; URL: https://github.com/emacs-evil/evil-collection
+;; Version: 0.0.1
+;; Package-Requires: ((emacs "25.1"))
+;; Keywords: emacs, tools, yaml
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+;;; Bindings for `yaml-mode'.
+
+;;; Code:
+(require 'evil-collection)
+(require 'yaml-mode nil t)
+
+(defconst evil-collection-yaml-mode-maps nil)
+
+(defun evil-collection-yaml-mode-set-evil-shift-width ()
+  "Set `evil-shift-width' according to `yaml-indent-offset'."
+  (setq evil-shift-width yaml-indent-offset))
```

Comment:
It uses `yaml-indent-offset`, a declaration is required to pass CI.

``` elisp
(defvar yaml-indent-offset)
```

and `evil-shift-width` is not a buffer-local variable, use `setq-local` instead.